app/views/ai-league/PageLeagueTeachers.vue

```diff
@@ -0,0 +1,417 @@
+        } catch (e) {
+          // Default to global page
+          application.router.navigate('league', { trigger: true })
+          return
+        }
+
+        if (['school-network', 'school-subnetwork', 'school-district'].includes(this.currentSelectedClan?.kind)) {
+          this.fetchChildClanDetails({ id: this.currentSelectedClan._id })
+            .catch(() => {
+              console.error('Failed to retrieve child clans.')
+            })
+        }
+        $.get('/esports/anonymous/' + this.currentSelectedClan._id).then((res) => {
+          this.anonymousPlayerName = res.anonymous
+        })
+
+        this.loadClanRequiredData({ leagueId: this.clanIdSelected })
+        this.loadChampionshipClanRequiredData({ leagueId: this.clanIdSelected })
+        this.loadCodePointsRequiredData({ leagueId: this.clanIdSelected })
+      } else {
+        this.loadGlobalRequiredData()
+        this.loadChampionshipGlobalRequiredData()
+        this.loadCodePointsRequiredData({ leagueId: '' })
+      }
+    },
+
+    inSelectedClan () {
+      if (!this.currentSelectedClan) {
+        return false
+      }
+
+      return (me.get('clans') || []).indexOf(this.currentSelectedClan._id) !== -1
+    },
+
+    isClanCreator () {
+      return (this.currentSelectedClan || {}).ownerID === me.id
+    },
+
+    isTeacher () {
+      return me.isTeacher()
+    },
+  },
+}
+</script>
+
+<template>
+  <main class="container">
+    <section class="row esports-header section-space">
+      <div class="col-sm-4">
+        <clan-selector
+          v-if="!isLoading && Array.isArray(myClans) && myClans.length > 0"
+          :clans="myClans"
+          :selected="clanIdSelected || clanIdOrSlug"
+          style="margin-bottom: 40px;"
+          @change="e => changeClanSelected(e)"
+        />
+      </div>
+    </section>
+
+    <section class="row">
+      <div class="col-lg-3 video-image-container">
+        <base-cloudflare-video
+          ref="video"
+          class="base-coludflare-video"
+          video-cloudflare-id="0bdb79d2ce155d589745f891b087f572"
+          :sound-on="false"
+          preload="none"
+          :loop="false"
+          :autoplay="false"
+          :controls="true"
+        />
+      </div>
+      <div class="col-lg-6">
+        <h1 class="text-h1">
+          {{ $t('league.codecombat_ai_league') }}
+        </h1>
+        <p class="text-p">
+          {{ $t('league.codecombat_ai_league_description') }}
+        </p>
+      </div>
+      <div class="col-lg-3 resources">
+        <a
+          class="resources__item screen"
+          href="https://docs.google.com/presentation/d/1ouDOu2k-pOxkWswUKuik7CbrUCkYXF7N_jNjGO0II6o/edit#slide=id.gb06b5c7fa4_0_10"
+          target="_blank"
+        >
+          {{ $t('league.teacher_getting_started_guide') }}
+        </a>
+        <a
+          class="resources__item view-exemplar"
+          href="https://codecombat.com/play/ladder/storm-siege"
```

Comment:
we should show the current one instead of hardcoding. @smallst can you help Adam here?

Reply:
oh sure. in app core utils we have activeArenas and related function which can point to current active ai-league.